src/containerapp/azext_containerapp/custom.py

```diff
@@ -5149,7 +5135,7 @@ def patch_apply_handle_input(cmd, patch_check_list, method, pack_exec_path):
 def patch_cli_call(cmd, resource_group, container_app_name, container_name, target_image_name, new_run_image, pack_exec_path):
     try:
         logger.warning("Applying patch for container app: " + container_app_name + " container: " + container_name)
-        subprocess.run(f"{pack_exec_path} rebase -q {target_image_name} --run-image {new_run_image}", shell=True, check=True)
+        subprocess.run(f"{pack_exec_path} rebase -q {target_image_name} --run-image {new_run_image} --force", shell=True, check=True)
```

Comment:
Could you please clarify why we need `--force` now? What has changed?

Reply:
I'll let @harryli0108 provide additional context, but AFAIK, the `--force` flag is now needed for the `pack rebase` command base on some of the latest changes made to their lifecycle to support our specific scenario.

Reply:
The `--force` is introduced because we couldn't include new releases into the image metadata. Pack rebase checks if the new run image is included in the existing image's metadata. This requires us to force rebasing the image. Otherwise, the rebase will fail with the following error: 
```
failed to rebase: rebase app image: new base image 'mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0.7-cbl-mariner2.0' not found in existing run image metadata: {"image":"mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0-cbl-mariner2.0"}